build.sbt

```diff
@@ -15,6 +15,10 @@ spAppendScalaVersion := true
 libraryDependencies += "org.apache.spark" %% "spark-core" % sparkVersion.value % "provided"
 libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.5" % "test"
```

Comment:
Hi Rishitesh. Better to just remove the dependency here?

Reply:
Yes. I missed that line. Thought its a transitive dependency

Reply:
Done and pushed.